site/gatsby-site/cypress/e2e/integration/submit.cy.js

```diff
@@ -1203,7 +1203,7 @@ describe('The Submit form', () => {
           'Sit quo accusantium quia assumenda. Quod delectus similique labore optio quaease\n',
         source_domain: `test.com`,
         editor_dissimilar_incidents: [5],
-        editor_similar_incidents: [16],
+        editor_similar_incidents: [321],
```

Comment:
I believe this will break prod since I had to hotfix this last week. I think the staging DB may need an update to prod's DB.

Reply:
I just did, running tests again.